### PR TITLE
Enable private tests in CI

### PR DESCRIPTION
**Pull request by je-cook**
## Description

Enable private tests, may have never been enabled by accident

## Interface Changes

N/A

## Checklist

I confirm that I have completed the following checks:

- [x] Tests run locally and pass `pytest tests --reactor`
- [X] Code quality checks run locally and pass `flake8` and `black .`
- [X] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

**Comment by je-cook** [comment]
This is currently missing two tests (I get 679 running locally)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -96,15 +101,15 @@ jobs:
       run: |
         pytest --cov=BLUEPRINT \
                --cov-report html:htmlcov_BLUEPRINT \
-               tests/BLUEPRINT --reactor
+               tests/BLUEPRINT --reactor --private
```
Comment (CoronelBuendia) [inline .github/workflows/ci.yml:104]: don't think we're going to need any BLUEPRINT private tests? Also seems to be failing atm

Comment (je-cook) [reply .github/workflows/ci.yml:104]: no it was just to make the testing fail sooner, I'll remove when it works

**Comment by je-cook** [comment]
This is not the cleanest fix in the world but I couldnt work out how to clone private data into the parent directory of the current working directory in one step

**Review by CoronelBuendia — APPROVED** [review]
Puah.. if it works, it works :)

**Pull request merged**